### DISCUSSION TITLE
Evento AusGames (round 6)

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#1022] [Ideia]
### Servidor afetado

Todos

### Qual o propósito da ideia?

Conforme o que foi divulgado no Patch Notes de melhorias AusTv, este evento visa aumentar a interação da STAFF dentro do jogo com um evento diferente, fora do horário. Vai ser divertido e compensador a todos os jogadores.

### Descrição da ideia

**Recomendação**: Esse evento tem muitos detalhes que podem ficar confusos se explicados só pelo chat. 
Por isso, é melhor realizar no chat de voz **Palco no Discord do servidor** (onde só quem tem permissão pode falar). 
**Isso cria proximidade da Staff com os players**, facilita para tirar dúvidas e **torna tudo mais divertido**.
Para quem não puder entrar no Discord, as informações podem ser disparadas no chat do jogo ao mesmo tempo.

# Aus Games

![Ideia](https://github.com/user-attachments/assets/46e5411e-1e88-45cc-9b46-95872844c7c7)

**Inspirado na série da Netflix Round 6, a ideia é organizar um evento com brincadeiras no mesmo estilo.** 

Os administradores serão os responsáveis por comandar as atividades, enquanto os ajudantes e voluntários (caso seja necessário) atuarão como "capangas" auxiliando.

Todos os jogadores competirão por um grande prêmio: **10 milhões de money no servidor escolhido + 5 mil de cash.** 
**(Antes do evento final os players podem decidir por dividir o prêmio ou ir para o último desafio)**

Os participantes que não vencerem receberão recompensas em forma de chaves, com raridade baseada na quantidade de jogos que participarem:

1 jogo: X chaves raras
2 ou 3 jogos: X chaves raras + X chave épica
4 ou 5 jogos: X chaves épicas

**Ao final todos os participantes recebem o cosmético do evento em seu guarda-roupa (Mascara do capanga)**

# Brincadeiras do Evento

## Batatones Frito 1, 2, 3

- Os jogadores precisam atravessar a arena enquanto uma música toca. Quando a música parar, quem se mexer será eliminado.

![image](https://github.com/user-attachments/assets/f986bde2-b8f9-403a-98e6-183a6cc5960f)

## Quarto Escuro

Os sobreviventes do desafio anterior precisam fugir dos monstros (ajudantes). 
Os jogadores ficarão cegos e lentos, enquanto os monstros terão força (hit kill) e lentidão .

![image](https://github.com/user-attachments/assets/83ea3242-c428-4d98-9132-60cb9e5a996c)

## Jogo das Frutas

Em uma sala com 9 plataformas, será mostrada uma fruta. 
Os jogadores precisam correr para a plataforma correspondente. Quem não chegar a tempo cai para a morte.

![image](https://github.com/user-attachments/assets/9b457a26-5948-4088-bcbf-66f0bf130344)

## Ponte de Cristal

Os jogadores devem atravessar uma ponte de vidro. Apenas algumas plataformas são seguras, escolher errado fará o vidro quebrar, levando à eliminação.

![image](https://github.com/user-attachments/assets/0722a20c-73a5-453e-a4ca-f29091e3b408)

## Quando as Luzes se Apagam...

Para adicionar emoção, as luzes podem apagar em qualquer momento entre as brincadeiras, especialmente se houver muitos jogadores ainda na disputa.
Ao apagar das luzes, o pvp é liberado durante 3 min, quem sobreviver continua.

![image](https://github.com/user-attachments/assets/205339bf-8d56-4f9d-a9b1-2924347caece)

## Futebol de Galinha

Uma atividade em equipe, inspirada nas regras da olimpíada.

![image](https://github.com/user-attachments/assets/7b9e7040-a475-4c1d-ae7f-c2b6466e410a)

## Finais Alternativos
(**Ao final os players podem decidir por dividir o prêmio ou ir para o último desafio**)

- Duelo Final: **Caso restem 2 jogadores**, eles se enfrentarão em um duelo.
![image](https://github.com/user-attachments/assets/21ef46cb-0407-4d08-963d-dadbf289c5b6)

- Batalha Final: **Se restarem 3 ou mais jogadores**, a final será um sumô para decidir o vencedor.

## Referências

- **Série da Netflix Round 6**

- **Vídeo 1** (https://www.youtube.com/watch?v=lw-cAdbRBz8)

- **Vídeo 2** (https://www.youtube.com/watch?v=-9_ug2EVuFk&list=PLs4RCIAQX-HPss_ueJ1rKKblGvpAVh3G2)

**Comment by ghost** [comment]
Se a ideia for aprovada, pf avisa quando começar a ser elaborada. Podemos preparar alguns materiais de divulgação do evento.

**Comment by leogianfagna** [comment]
@Yokiro27 podemos fazer isso para o aniversário da aus, 6/2. o que acha?

**Reply by ghost** [reply]
Claro, vai ser ótimo
Acha necessário acrescentar algo que remeta ao tema de aniversário?

**Reply by leogianfagna** [reply]
se tiver ideia pode colocar. mas jogos desses já é um bom negocio
